compass/landice/tests/humboldt/mesh.py

```diff
@@ -0,0 +1,347 @@
+                'gis_1km_preCull.nc', '-m'
+                'distance', '-d', '50.0']
+
+        check_call(args, logger=logger)
+
+        print('calling set_lat_lon_fields_in_planar_grid.py')
+        args = ['set_lat_lon_fields_in_planar_grid.py', '-f',
+                'gis_1km_preCull.nc', '-p', 'gis-gimp']
+
+        check_call(args, logger=logger)
+
+        print('calling MpasMaskCreator.x')
+        args = ['MpasMaskCreator.x', 'gis_1km_preCull.nc',
+                'humboldt_mask.nc', '-f', 'Humboldt.geojson']
+
+        check_call(args, logger=logger)
+
+        print('culling to geojson file')
+        dsMesh = xarray.open_dataset('gis_1km_preCull.nc')
+        humboldtMask = xarray.open_dataset('humboldt_mask.nc')
+        dsMesh = cull(dsMesh, dsInverse=humboldtMask, logger=logger)
+        write_netcdf(dsMesh, 'humboldt_culled.nc')
+
+        print('Marking horns for culling')
+        args = ['mark_horns_for_culling.py', '-f', 'humboldt_culled.nc']
+        check_call(args, logger=logger)
+
+        print('culling and converting')
+        dsMesh = xarray.open_dataset('humboldt_culled.nc')
+        dsMesh = cull(dsMesh, logger=logger)
+        dsMesh = convert(dsMesh, logger=logger)
+        write_netcdf(dsMesh, 'humboldt_dehorned.nc')
+
+        print('calling create_landice_grid_from_generic_MPAS_grid.py')
+        args = ['create_landice_grid_from_generic_MPAS_grid.py', '-i',
+                'humboldt_dehorned.nc', '-o',
+                'Humboldt_1to10km.nc', '-l', levels, '-v', 'glimmer',
+                '--beta', '--thermal', '--obs', '--diri']
+
+        check_call(args, logger=logger)
+
+        print('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'humboldt_1km_2020_04_20.epsg3413.icesheetonly.nc',
+                '-d', 'Humboldt_1to10km.nc', '-m', 'b', '-t']
+        check_call(args, logger=logger)
+
+        print('Marking domain boundaries dirichlet')
+        args = ['mark_domain_boundaries_dirichlet.py',
+                '-f', 'Humboldt_1to10km.nc']
+        check_call(args, logger=logger)
+
+        print('calling set_lat_lon_fields_in_planar_grid.py')
+        args = ['set_lat_lon_fields_in_planar_grid.py', '-f',
+                'Humboldt_1to10km.nc', '-p', 'gis-gimp']
+        check_call(args, logger=logger)
+
+        print('creating graph.info')
+        make_graph_file(mesh_filename='Humboldt_1to10km.nc',
+                        graph_filename='graph.info')
+
+    def build_cell_width(self):
+
+        # get needed fields from GIS dataset
+        f = netCDF4.Dataset('greenland_8km_2020_04_20.epsg3413.nc', 'r')
+        f.set_auto_mask(False)  # disable masked arrays
+
+        x1 = f.variables['x1'][:]
+        y1 = f.variables['y1'][:]
+        thk = f.variables['thk'][0, :, :]
+        topg = f.variables['topg'][0, :, :]
+        vx = f.variables['vx'][0, :, :]
+        vy = f.variables['vy'][0, :, :]
+
+        # subset data - optional
```

Comment:
Add a note elaborating why this might be done.

Reply:
We could probably remove this. It's legacy code that was never used, and it's potentially dangerous if inadvertently changed.